charts/landscaper/Chart.yaml

```diff
@@ -19,9 +19,9 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.0
+version: v0.5.3
```

Comment:
The chart version can stay at 0.1.0 only the appVersion should be increased

Reply:
Changed for all Helm charts and also in the `update-helm-chart-version.sh` script.